Jenkinsfile

```diff
@@ -71,7 +71,7 @@ pipeline {
           script {
             try {
               sh '''
-                git fetch --no-tags --progress git@github.com:intellihr/ui-components.git \
+                git fetch --no-tags --progress git@github.com:intelliHR/ui-components.git \
```

Comment:
why did this change?

Reply:
I don't care as long as it doesn't break anything

Reply:
the github pages deploy was broken, this should fix it.